### PR TITLE
User home and profile display

### PR DESCRIPTION
**Pull request by inahsanjuan**

### CODE REVIEW

File: app/Http/Controllers/Auth/AuthController.php [hunk 1]
```diff
@@ -69,4 +69,8 @@ protected function create(array $data)
             'password' => bcrypt($data['password']),
         ]);
     }
+
+    public function logintest() {
```
Comment (nguyenthanhtung88) [inline app/Http/Controllers/Auth/AuthController.php]: Why it is login test not real login 😄

Comment (inahsanjuan) [inline app/Http/Controllers/Auth/AuthController.php]: I was going to remove that. Oops.

File: app/Http/Controllers/User/MemberController.php [hunk 2]
```diff
@@ -16,7 +16,8 @@ class MemberController extends Controller
      */
     public function index()
     {
-        return view('user.members');
+        $members = User::all();
+        return view('user.members', compact('members'));
```
Comment (nguyenthanhtung88) [inline app/Http/Controllers/User/MemberController.php]: It make sense if you use `members.index` view, please try to use view name same as action in controller.
It help us to find view and modify without accessing to controller to see what view used 🙇

File: app/Http/Controllers/User/MemberController.php [hunk 3]
```diff
@@ -48,7 +49,8 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $user = User::find($id);
+        return view('user.profile', compact('user'));
```
Comment (nguyenthanhtung88) [inline app/Http/Controllers/User/MemberController.php]: Please use view `member.show`

File: app/Http/Controllers/User/UserController.php [hunk 4]
```diff
@@ -47,9 +47,10 @@ public function store(Request $request)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function show($id)
+    public function show()
     {
-        //
+        $user = Auth::user();
+        return view('user.home', compact('user'));
```
Comment (nguyenthanhtung88) [inline app/Http/Controllers/User/UserController.php]: Please use view `user.show`

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -1,65 +1,58 @@
...
-                        </div>
+@section('content')
+    {{ Form::open(['url' => '/login', 'class' => 'form-horizontal']) }}
+        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+            {{ Form::label('email', trans('auth.email'), ['class' => 'col-md-4 control-label']) }}
+
+            <div class="col-md-4">
+                {{ Form::email('email', old('email'), ['class' => 'form-control']) }}
+
+                @if ($errors->has('email'))
```
Comment (nguyenthanhtung88) [inline resources/views/auth/login.blade.php]: Oops, i saw same HTML block code duplicate in your blade views:

- https://github.com/framgia/fels_168/pull/6/files#diff-731fd6d11c0d36b0d7d6853b53a2b716R29
- https://github.com/framgia/fels_168/pull/6/files#diff-79246e6ca638fe8b51af4c413e29323cR15
- ...
The difference is only the `key`.

Comment (inahsanjuan) [inline resources/views/auth/login.blade.php]: Yes that is much better, thanks.

**Comment by nguyenthanhtung88** [comment]
OK

**Pull request merged**